src/components/dialogs/create-contingency-list-dialog.js

```diff
@@ -54,6 +66,14 @@ export const CreateContingencyListDialog = ({ open, onClose }) => {
     const classes = useStyles();
     const intl = useIntl();
     const timer = React.useRef();
+    const [isConfirmationPopupOpen, setOpenConfirmationPopup] = useState(false);
+    const [currentScript, setCurrentScript] = useState(null);
+    const currentCriteriaBasedFilter = useRef(null);
+    const [
+        currentCriteriaBasedFilterEquipmentTypeSelected,
+        setCurrentCriteriaBasedFilterEquipmentTypeSelected,
+    ] = useState(false);
```

Comment:
useState(null) ?

Reply:
Ah, I thought it was the type here, but it's only a boolean to say if it's selected or not.

Reply:
I will rename this variable to clarify that.